tests/Oryx.BuildImage.Tests/BenvTest.cs

```diff
@@ -66,7 +66,7 @@ public void PipelineTestInvocationVsoFocal()
         {
             var imageTestHelper = new ImageTestHelper();
             BuildImagesHaveOryxPathsEnvironmentVariableAvailable(
-                imageTestHelper.GetVsoBuildImage("vso-focal"));
+                imageTestHelper.GetVsoBuildImage(ImageTestHelperConstants.VsoUbuntu));
```

Comment:
consider renaming `ImageTestHelperConstants.VsoUbuntu` -> `ImageTestHelperConstants.VsoFocal`, which is consistent with `ImageTestHelperConstants.LtsVersionsStretch`